README.md

```diff
@@ -7,20 +7,41 @@
 [![Coverity Scan Build Status](https://scan.coverity.com/projects/14936/badge.svg)](https://scan.coverity.com/projects/itsallcode-openfasttrace)
 [![Download](https://api.bintray.com/packages/itsallcode/itsallcode/openfasttrace/images/download.svg) ](https://bintray.com/itsallcode/itsallcode/openfasttrace/_latestVersion)
 
-![logo](src/main/resources/openfasttrace_logo.svg)
+## Installation
 
-# Installation
+### Dependencies
 
-## Dependencies
+#### Runtime Dependencies
 
-OpenFastTrace needs a Java JDK 8 (or later) implementation and Maven as build tool.
+OpenFastTrace only needs a Java 8 (or later) runtime environment to run.
 
-## On Linux
+#### Build time Dependencies
 
-### Ubuntu or Debian
+At build time the following dependencies are additionally required.
+
+| Dependency                                                                   | Purpose                                                | License                       |
+-------------------------------------------------------------------------------|--------------------------------------------------------|--------------------------------
+| [Apache Maven](https://maven.apache.org/)                                    | Build tool                                             | Apache License 2.0            |
+| [Equals Verifier](https://github.com/jqno/equalsverifier)                    | Automatic contract checker for `equals()` and `hash()` | Apache License 2.0            |
+| [Hamcrest Auto Matcher](https://github.com/itsallcode/hamcrest-auto-matcher) | Speed-up for building Hamcrest matchers                | GNU General Public License V3 |
+| [JUnit](https://junit.org/junit4/index.html)                                 | Unit testing framework                                 | Eclipse Public License 1.0    |
+| [Mockito](http://site.mockito.org/)                                          | Mocking framework                                      | MIT License                   |
+| [System rules](https://stefanbirkner.github.io/system-rules/)                | JUnit rules for testing `STDOUT`, `STDIN` and `STDERR` | Common Public License 1.0     |
```

Comment:
This looks like you need to install all these libs. But actually you only need to install maven. Maybe make separate tables with build tools and libs? The next sections describes what is really required by I was confused at first.